app/lib/actions/katello/content_view_version/incremental_update.rb

```diff
@@ -166,8 +166,9 @@ def generate_description(version, content)
             end
             humanized_lines << ''
           end
-          version.description = humanized_lines.join("\n")
-          version.save!
+          content_view = version.content_view
+          content_view.description = humanized_lines.join("\n")
+          version.save! && content_view.save!
```

Comment:
The [current description method](https://github.com/Katello/katello/blob/master/app/models/katello/content_view_version.rb#L108) is read only (no `def description=`), and it gets `history.publish.successful.first.try(:notes)`. Perhaps we should [1] revert the change I made in `incremental_update.rb` and [2] add a `def description=` method in `content_view_version.rb` that overwrites the notes field if the `ContentViewHistory` object exists.

If we want to support appending or checking the description is blank, it should be done from the `incremental_update.rb` so the `description=` method isn't confusing to later contributors. The only question is (to be answered by @daviddavis or @jlsherrill) should we (a) append or (b) replace only if empty?